tpm_quote/src/lib.rs

```diff
@@ -0,0 +1,107 @@
+use error::Error;
+use key::{get_key_from_persistent_handle, TPM_AK_NVINDEX};
+use std::convert::TryInto;
+use tss_esapi::{
+    interface_types::algorithm::HashingAlgorithm,
+    structures::{
+        Attest, AttestInfo, PcrSelectionListBuilder, PcrSlot, Public, QuoteInfo, Signature,
+    },
+    traits::{Marshall, UnMarshall},
+    Context,
+};
+use verify::verify_quote_signature;
+
+pub mod command;
+pub mod convert;
+pub mod error;
+pub mod key;
+pub mod verify;
+
+#[allow(clippy::type_complexity)]
+pub fn get_quote(
+    context: &mut Context,
+    pcr_list: &[u8],
+    nonce: Option<&[u8]>,
+) -> Result<(Vec<u8>, Vec<u8>, Vec<u8>), Error> {
```

Comment:
Add a comment to help understanding the return type?

Reply:
See 44184e424d7c2f74b6cb058a6ac77438c4651445.